.travis.yml

```diff
@@ -45,6 +51,7 @@ matrix:
 
   allow_failures:
     - php: hhvm
+    - env: PHPSTAN=1
```

Comment:
For a failure list to work you also need to include the php version each time afaik.